template/src/state/slices/authSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { loginFulfilled } from 'services/auth';
+
+const initialState = {
+  authenticated: false,
+  user: {},
+  info: {},
+};
+
+const authSlice = createSlice({
+  name: 'auth',
+  initialState,
+  extraReducers: builder => {
+    builder.addMatcher(loginFulfilled, (state, { payload: { data, info } }) => {
```

Comment:
How do we reset the state on the reducer if we need it?

Reply:
To reset the state we should create a simple action that returns initialState. In this case 

```suggestion
  initialState,
  reducers: 
    reset() {
      return initialState;
    },
  extraReducers: builder => {
```